### ISSUE TITLE
Request: remember export options

### ISSUE THREAD

**Issue by alanlorence** [videovillage/Screen-Issues#50]
Would be so helpful if export options were remembered. Some may be now, but "Sizing" is not -- it always defaults to "Current" so I have to change it every time. Especially annoying when using "Width" or "Height" and also have to retype the value every time.

**Comment by dylanstoel** [comment]
I totally agree with this. I make proxies everyday and redoing this everyday starts to irritate a bit haha
